docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
```

Comment:
`Minikube` is not being linked up to the minikube repo correctly. Perhaps we can just use the `[Minikube](https://github.com/kubernetes/minikube)` notation